prisma/schema.prisma

```diff
@@ -56,6 +56,8 @@ model Post {
   id       Int       @id @default(autoincrement())
   content  String
   userId   Int
+  createdAt DateTime    @default(now())
+  updatedAt DateTime    @updatedAt
```

Comment:
have you tested running this migration on a table that didn't ahve these 2 columns but had existing rows? i think the migration will probably break on the updatedAt.

I think you need to create 2 migrations:

MODEL for first migration:

```
createdAt DateTime    @default(now())
  updatedAt DateTime    @default(now()) // set an initial value for all updatedAt for each existing row
```

generate the migration file for the above model, THEN

change model:

```
createdAt DateTime    @default(now())
  updatedAt DateTime    @updatedAt
  
  ```
 now generate a second migration, so all previous existing rows now have a default value and new rows will get the correct updatedAt

Reply:
fixed
it has two migrations now

Reply:
but the first elements breaks the code though